### PR TITLE
[i18n] Update Spanish translation

### PR DESCRIPTION
**Pull request by fitojb**

### CODE REVIEW

**Comment by battlemidget** [comment]
@fitojb Hi! Thank you for the PR, would you mind adding your DCO? I'm not sure if a commit summary is actually required for translations, but, I'll leave that up to @bmr-cymru

**Comment by fitojb** [comment]
@battlemidget If you’re referring to a “developer certificate of origin”, I have already signed off the commit message, as stated in the contribution guidelines at this repo’s wiki. Is there anything else I need to do? :)

**Comment by bmr-cymru** [comment]
Sorry - I had a quick look when this arrived, and meant to add labels etc. - I think the commit looks fine, but I wanted to get a 2nd opinion on some of the translations (my Spanish is "learner/basic" ;) ). So far all the ones I checked are a genuine improvement so I think we can merge the whole set shortly.

File: po/es.po [hunk 1]
```diff
@@ -30,128 +29,123 @@ msgstr "sosreport (versión %s)"
 #: ../sos/sosreport.py:977
 #, python-format
 msgid "plugin %s does not validate, skipping"
-msgstr "no se validó el plugin %s, ignorándolo"
+msgstr "el complemento %s no se valida; se ha omitido"
```
Comment (ealcaniz) [inline po/es.po]: +msgstr "el complemento %s no se validado; se omite"

File: po/es.po [hunk 2]
```diff
@@ -30,128 +29,123 @@ msgstr "sosreport (versión %s)"
...
 
 #: ../sos/sosreport.py:979
-#, fuzzy
 msgid "does not validate"
-msgstr "no se validó el plugin %s, ignorándolo"
+msgstr "no se valida"
```
Comment (ealcaniz) [inline po/es.po]: +msgstr "no se ha validado"

File: po/es.po [hunk 3]
```diff
@@ -30,128 +29,123 @@ msgstr "sosreport (versión %s)"
...
+#, python-format
 msgid "plugin %s does not install, skipping: %s"
-msgstr "no se instaló el plugin %s, ignorándolo"
+msgstr "el complemento %s no se puede instalar; se ha omitido: %s"
 
 #: ../sos/sosreport.py:1027
 #, python-format
 msgid "Unknown or inactive profile(s) provided: %s"
-msgstr ""
+msgstr "Se proveyeron perfiles desconocidos o inactivos: %s"
```
Comment (ealcaniz) [inline po/es.po]: +msgstr "Los perfiles proporcionados son desconocidos o inactivos: %s"

File: po/es.po [hunk 4]
```diff
@@ -30,128 +29,123 @@ msgstr "sosreport (versión %s)"
...
-#, fuzzy
 msgid "no valid profiles found"
-msgstr "no se encontró un plugin válido"
+msgstr "no se encontró ningún perfil válido"
 
 #: ../sos/sosreport.py:1174
-#, fuzzy
 msgid "The following profiles are available:"
-msgstr "Las siguientes opciones del plugin están disponibles:"
+msgstr "Los perfiles siguientes están disponibles:"
```
Comment (ealcaniz) [inline po/es.po]: +msgstr "Los siguientes perfiles están disponibles:"

File: po/es.po [hunk 5]
```diff
@@ -30,128 +29,123 @@ msgstr "sosreport (versión %s)"
...
 
 #: ../sos/sosreport.py:1197
 msgid "Press ENTER to continue, or CTRL-C to quit.\n"
 msgstr "Presione INTRO para continuar o CTRL-C para salir.\n"
 
 #: ../sos/sosreport.py:1216
-#, fuzzy
 msgid " Setting up archive ..."
-msgstr "Encriptando el archivo..."
+msgstr " Configurando el archivador…"
```
Comment (ealcaniz) [inline po/es.po]: +msgstr " Configurando el archivo …"  but what means? is this backup?

File: po/es.po [hunk 6]
```diff
@@ -30,128 +29,123 @@ msgstr "sosreport (versión %s)"
...
 #: ../sos/sosreport.py:1250
 msgid " Setting up plugins ..."
-msgstr ""
+msgstr " Configurando los complementos…"
 
 #: ../sos/sosreport.py:1282
-#, fuzzy
 msgid " Running plugins. Please wait ..."
-msgstr "Ejecutando complementos. Por favor espere..."
+msgstr " Ejecutando los complementos. Espere…"
```
Comment (ealcaniz) [inline po/es.po]: +msgstr " Ejecutando los complementos. Espere por favor …"

File: po/es.po [hunk 7]
```diff
@@ -30,128 +29,123 @@ msgstr "sosreport (versión %s)"
...
 
 #: ../sos/sosreport.py:1490
 msgid "Creating compressed archive..."
-msgstr "Creando un archivo comprimido..."
+msgstr "Creando un archivador comprimido…"
```
Comment (ealcaniz) [inline po/es.po]: +msgstr "Creando el archivo comprimido …"

**Comment by ealcaniz** [comment]
Please correct the spanish language and I will double check when it is resolve it. 
My question is 
 msgid " Setting up archive ..."
-msgstr "Encriptando el archivo..."
+msgstr " Configurando el archivador…"
      ealcaniz
ealcaniz just now
+msgstr " Configurando el archivo …" but what means? is this backup?

if means we are archiving the configuration as backup??

**Comment by bmr-cymru** [comment]
It *should* mean something like "preparing the archive", or "making the archive ready" - it is printed before we create the temporary directories where the report will be built (internally in `sos` this is known as the `Archive`, specifically it's an instance of a `FileCacheArchive`.

**Comment by ealcaniz** [comment]
Ok. 

+msgstr "Preparando el fichero…"

**Comment by ealcaniz** [comment]
@pmoravec thanks for involving me. I am helping to translate OpenStack in Spanish and Catalan too. Ping me with any help I could offer.

File: po/es.po (hunk 5 above)
Comment (iranzo) [inline po/es.po]: Extra space in front of string

File: po/es.po (hunk 6 above)
Comment (iranzo) [inline po/es.po]: Same extra space

File: po/es.po (hunk 7 above)
Comment (iranzo) [inline po/es.po]: I would leave 'archivo'

File: po/es.po (hunk 1 above)
Comment (fitojb) [inline po/es.po]: I don’t think I’ll mirror the gerund you’re suggesting.

File: po/es.po (hunk 5 above)
Comment (fitojb) [inline po/es.po]: @iranzo The extra space is also in the source string. That should be dealt with in a separate PR.

File: po/es.po (hunk 6 above)
Comment (fitojb) [inline po/es.po]: @ealcaniz It’s not proper to abuse “Please” in Spanish translations. The “usted” treatment makes up for it.

File: po/es.po (hunk 7 above)
Comment (fitojb) [inline po/es.po]: @ealcaniz @iranzo GNOME uses “archivador”, though.

File: po/es.po (hunk 2 above)
Comment (fitojb) [inline po/es.po]: Will do.

File: po/es.po (hunk 3 above)
Comment (fitojb) [inline po/es.po]: I don’t like the passive voice, which isn’t present in the original string.

**Comment by fitojb** [comment]
Pull request updated.

**Comment by fitojb** [comment]
Is anything left to do here? =)

**Comment by bmr-cymru** [comment]
Not much I don't think - last time I checked in here there was still a lot of discussion going on, but if you guys are happy with what we have in the PR now then we can get that merged shortly.

**Comment by ealcaniz** [comment]
+1 approved too
